src/router/routes/judging.ts

```diff
@@ -208,6 +216,82 @@ export class JudgingController extends ParentRouter implements IExpressControlle
         }
     }
 
+    /**
+    * @api {post} /judging/score Update a Scoring
+    * @apiVersion 2.0.0
+    * @apiName Update Scoring
+    * @apiPermission TeamMemberPermission
+    * @apiUse AuthArgumentRequired
+    * @apiGroup Judging
+    * @apiParam {Number} project_id The project's uid
+    * @apiParam {String} judge The email of the judge
+    * @apiParam {Number} creativity Score for the 'Creativity' category
+    * @apiParam {Number} technical Score for the 'Technical' category
+    * @apiParam {Number} implementation Score for the 'Implementation' category
+    * @apiParam {Number} clarity Score for the 'Clarity' category
+    * @apiParam {Number} growth Score for the 'Growth' category
+    * @apiParam {Boolean} submitted Whether this scoring is finished
+    * @apiParam {Number} [humanitarian] Score for the 'Humanitarian' award
+    * @apiParam {Number} [supply_chain] Score for the 'Supply Chain' award
+    * @apiParam {Number} [environmental] Score for the 'environmental' award
+    * @apiSuccess {Score} data The updated score
+    * @apiUse IllegalArgumentError
+    * @apiUse ResponseBodyDescription
+    */
+     private async updateScoreHandler(req: Request, res: Response, next: NextFunction) {
+        if (!req.body) {
+            return next(new HttpError('Could not find request body', 400));
+        }
+        
+        let score: Score;
+        try {
+            score = new Score(req.body);
+        } catch (error) {
+            return next(new HttpError('Some properties were not as expected.', 400));
+        }
+
+        try {
+            const result = await this.scoreDataMapper.update(score);
+            return this.sendResponse(res, new ResponseBody('Success', 200, result));
+        } catch (error) {
+            return Util.errorHandler500(error, next);
+        }
+    }
+
+    /**
+    * @api {post} /judging/score Delete a Scoring
+    * @apiVersion 2.0.0
+    * @apiName Delete Scoring
+    * @apiPermission TeamMemberPermission
+    * @apiUse AuthArgumentRequired
+    * @apiGroup Judging
+    * @apiParam {Number} project_id The project's uid
+    * @apiParam {String} judge The email of the judge
+    * @apiSuccess {Score} data The deleted score
+    * @apiUse IllegalArgumentError
+    * @apiUse ResponseBodyDescription
+    */
+     private async deleteScoreHandler(req: Request, res: Response, next: NextFunction) {
+        if (!req.body) {
+            return next(new HttpError('Could not find request body', 400));
+        }
+        
+        if (!req.body.project_id || !parseInt(req.body.uid, 10)) {
```

Comment:
The second part of this should be !parseInt(req.body.project_id, 10) instead of !parseInt(req.body.uid)